shell/components/SortableTable/index.vue

```diff
@@ -374,8 +374,10 @@ export default {
       eventualSearchQuery = this.$route.query?.q;
     }
 
+    const isLoading = this.loading || false;
```

Comment:
Sometimes loading has a value and that's what we should initiate our state with.